Dockerfile

```diff
@@ -44,9 +44,11 @@ HEALTHCHECK --interval=5s --timeout=1s \
 FROM base AS dev
 ENV NODE_ENV=development
 
-COPY tsconfig.json \
+COPY jest.config.js \
+    tsconfig.json \
     ./
 COPY --from=npm-dev /app/ .
+COPY test/ test/
```

Comment:
not alphabetical in this case (`test` and `src`); not that it matters as any of the two will invalidate the caching layers from then on and they are both relatively fast operations

Reply:
I'm assuming that `src` changes more often that `test`.